apps/nextjs-website/src/components/HowTo.tsx

```diff
@@ -0,0 +1,39 @@
+'use client';
+import { HowToProps } from '@pagopa/pagopa-editorial-components/dist/components/HowTo';
+import { HowTo as HowToEC } from '@pagopa/pagopa-editorial-components';
+import * as MuiIcons from '@mui/icons-material';
+import { HowToSection } from '@/lib/fetch/types/PageSection';
+
+const makeHowToProps = ({
+  rowMaxSteps,
+  link,
+  steps,
+  ...rest
+}: HowToSection): HowToProps => ({
+  ...(rowMaxSteps && { rowMaxSteps }),
+  ...(link && {
+    link: {
+      href: link.href,
+      label: link.text ?? link.href,
+    },
+  }),
+  steps: steps.map((step) => ({
+    title: step.title,
+    description: step.description, // TODO: Parse rich text (markdown)
+    ...(step.icon && {
+      stepIcon: {
+        icon: step.icon as keyof typeof MuiIcons,
```

Comment:
I would solve it as it has been resolved in [other parts of the code](https://github.com/pagopa/b2b-portals/blob/fda5d66/apps/nextjs-website/src/components/PreHeader.tsx#L37)